### PR TITLE
Two2three

### PR DESCRIPTION
**Pull request by hlasimpk**
Changes to make SIMBAD python3 compatible.

### CODE REVIEW

File: simbad/command_line/simbad_database.py [hunk 1]
```diff
@@ -335,7 +340,7 @@ def create_contaminant_db(database, add_morda_domains, nproc=2, submit_qtype=Non
                 if os.path.isfile(output):
                     simbad.db.convert_pdb_to_dat(output, final)
                 else:
-                    print "File missing: {}".format(output)
+                    print("File missing: {}".format(output))
```
Comment (fsimkovic) [inline simbad/command_line/simbad_database.py]: log.warning

Comment (fsimkovic) [inline simbad/command_line/simbad_database.py]: log.warning

File: simbad/util/__init__.py [hunk 2]
```diff
@@ -55,6 +56,11 @@ def output_files(run_dir, result, output_pdb, output_mtz):
     shutil.copyfile(input_mtz, output_mtz)
 
 
+def python_version():
+    """Return python version"""
+    return sys.version_info.major
```
Comment (fsimkovic) [inline simbad/util/__init__.py]: no, no, no, no. please just use wherever necessary
```
if sys.version_info.major < 3:
    # py2 import
else:
    # py3 import
```

File: simbad/util/simbad_results.py [hunk 3]
```diff
@@ -134,4 +134,4 @@ def get_files(self, search_type, num_results=10):
     SR = SimbadResults(args.work_dir)
 
     for fc in SR.top_files():
-        print "FC ", fc, fc.ref_log, fc.ref_pdb_annotation
+        print("FC ", fc, fc.ref_log, fc.ref_pdb_annotation)
```
Comment (fsimkovic) [inline simbad/util/simbad_results.py]: log.debug

File: simbad/util/tests/test_pdb_util.py [hunk 4]
```diff
@@ -5,19 +5,26 @@
...
 
 
 def internet_on():
     try:
-        urllib2.urlopen("http://www.google.com", timeout=1)
+        urlopen("http://www.google.com", timeout=1)
         return True
-    except urllib2.URLError:
+    except URLError:
         return False
```
Comment (fsimkovic) [inline simbad/util/tests/test_pdb_util.py]:
```
try:
    urlopen(...)
except URLError:
    return False
else:
    return True
```

File: simbad/util/__init__.py (hunk 2 above)
Comment (hlasimpk) [reply simbad/util/__init__.py]: Yeah I wasn't sure about this and I thought you'd be sure to correct me

File: simbad/util/simbad_results.py [hunk 5]
```diff
@@ -134,4 +134,4 @@ def get_files(self, search_type, num_results=10):
     SR = SimbadResults(args.work_dir)
 
     for fc in SR.top_files():
-        print("FC ", fc, fc.ref_log, fc.ref_pdb_annotation)
+        logger.debug("FC ", fc, fc.ref_log, fc.ref_pdb_annotation)
```
Comment (fsimkovic) [inline simbad/util/simbad_results.py]: won't work. you need to use `%s`

File: simbad/command_line/simbad_database.py [hunk 6]
```diff
@@ -340,7 +340,7 @@ def create_contaminant_db(database, add_morda_domains, nproc=2, submit_qtype=Non
                 if os.path.isfile(output):
                     simbad.db.convert_pdb_to_dat(output, final)
                 else:
-                    print("File missing: {}".format(output))
+                    logger.warning("File missing: {}".format(output))
```
Comment (fsimkovic) [inline simbad/command_line/simbad_database.py]: use `%s` formatting with comma-separated arguments

**Pull request merged**